src/base/bittorrent/torrentimpl.h

```diff
@@ -49,6 +49,16 @@
 #include "torrent.h"
 #include "torrentinfo.h"
 
+#if (LIBTORRENT_VERSION_NUM >= 20003)
```

Comment:
libtorrent didn't export `struct file_prio_alert` so you added this right?
Won't it cause problems after libtorrent fixed it? https://github.com/arvidn/libtorrent/commit/672beb857626b9246da210cd930afab965f5ef3f
Maybe use `LIBTORRENT_VERSION_NUM == 20003`?

Reply:
>libtorrent didn't export struct file_prio_alert so you added this right?

No. The forward declaration is missing in `libtorrent/fwd.hpp`.